src/catalogue/category/catalogueCategoryDialog.component.tsx

```diff
@@ -492,7 +492,9 @@ const CatalogueCategoryDialog = React.memo(
     return (
       <Dialog
         open={open}
-        onClose={(reason) => reason == 'backdropClick' ?? handleClose}
+        onClose={(event, reason) =>
+          event && reason == 'backdropClick' ? null : handleClose()
+        }
```

Comment:
```suggestion
        onClose={(_event, reason) =>
          reason !== 'backdropClick' && handleClose()
        }
```
This should also work without needing to include event which should never be empty here anyway. It also means if any other reasons are added in MUI in the future the logic will still work. Same for the others.